js/test/Ice/timeout/Client.ts

```diff
@@ -87,7 +87,7 @@ export class Client extends TestHelper {
         {
             const connection = await timeout.ice_getConnection();
             await controller.holdAdapter(-1);
-            await connection.close(Ice.ConnectionClose.GracefullyWithWait);
+            const closed = connection.close();
```

Comment:
We should fix the exception and simplify this code in a follow-up PR. See #2682.